src/modules/comment/comment.controller.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  HttpCode,
+  Param,
+  Post,
+  Get,
+} from '@nestjs/common';
+import { CommentService } from './comment.service';
+import { CreateCommentDto } from './dto/create-comment.dto';
+
+@Controller('comment')
+export class CommentController {
+  constructor(private readonly commentService: CommentService) {}
+
+  @Get()
+  findAll() {
+    return this.commentService.findAll();
+  }
+
+  @Get(':id')
+  findById(@Param('id') id: string) {
+    return this.commentService.findById(id);
+  }
+
+  @Post()
+  @HttpCode(201)
+  async create(@Body() createCommentDto: CreateCommentDto) {
+    return this.commentService.create(createCommentDto);
+  }
+
+  @Delete(':id')
+  remove(@Param('id') id: string) {
+    return this.commentService.delete(id);
```

Comment:
Mesma coisa aqui deleteComment como nome do service

Reply:
Não entendi essa parte. O método delete já está no contexto do comment .. chamar o método de deletelComment seria redundante, nao?

Reply:
seria pra seguir o do album, se for o caso entao deleta o que está no album de deleteAlbum

Reply:
Feito